src/networking/webSocket.ts

```diff
@@ -25,5 +25,15 @@ export function setupSocket(signaller: Signal) {
     signaller.trigger(EventType.ConnectionMade);
   });
 
+  const interval = setInterval(function ping() {
```

Comment:
I think these should be inside the `.on("connection"`, it's probably not valid to ping on an unopened connection.